openmc/mgxs_library.py

```diff
@@ -348,7 +398,7 @@ def num_delayed_groups(self, num_delayed_groups):
     @representation.setter
     def representation(self, representation):
 
-        # Check it is of valid type.
+        # Check it is of valid value.
```

Comment:
Might consider removing this comment altogether, I think it is self explanatory.